diff -urN Mesa-10.2.0-rc4.orig/configure.ac Mesa-10.2.0-rc4/configure.ac
--- Mesa-10.2.0-rc4.orig/configure.ac	2014-05-27 22:54:56.403180657 +0200
+++ Mesa-10.2.0-rc4/configure.ac	2014-05-27 22:56:47.353186229 +0200
@@ -1846,7 +1846,6 @@
             HAVE_GALLIUM_R300=yes
             PKG_CHECK_MODULES([RADEON], [libdrm_radeon >= $LIBDRM_RADEON_REQUIRED])
             gallium_require_drm_loader
-            gallium_require_llvm "Gallium R300"
             GALLIUM_DRIVERS_DIRS="$GALLIUM_DRIVERS_DIRS r300"
             gallium_check_st "radeon/drm" "r300/dri" "" "" ""
             DRICOMMON_NEED_LIBDRM=yes
